eventlog/el/main_test.go

```diff
@@ -20,7 +20,7 @@ func TestMain(m *testing.M) {
 	log.MainTest(m)
 }
 
-const blockInterval = 100 * time.Millisecond
+const blockInterval = 1 * time.Second
```

Comment:
I started to use a lot of these:

https://github.com/dedis/cothority/pull/1292/files#diff-61a6ae16d2daf74433eb76df236b74e0R555

To verify when the new instance is created. But I think we need to do the same with the `ClientTransaction` and make a way to get the result of a `ClientTransaction` from the node.

Reply:
Are you asking for a change here... I'm  not sure I see what you are proposing.

Reply:
I thought you were waiting for all the transactions to be included and pointed to how it should be done - polling each `interval` to see if the proof `match()`es.